egs/aishell/asr/simple_v1/aishell.py

```diff
@@ -0,0 +1,117 @@
+"""
+About the Aishell corpus
+Aishell is an open-source Chinese Mandarin speech corpus published by Beijing Shell Shell Technology Co.,Ltd.
+publicly availble on https://www.openslr.org/33
+"""
+
```

Comment:
Would you mind writing the author copyright info here, i.e., your name and your company?

Reply:
Done

Reply:
thanks